tpl/openapi/openapi3/openapi3.go

```diff
@@ -90,7 +93,26 @@ func (ns *Namespace) Unmarshal(r resource.UnmarshableResource) (*OpenAPIDocument
 			return nil, err
 		}
 
-		err = kopenapi3.NewLoader().ResolveRefsIn(s, nil)
+		loader := kopenapi3.NewLoader()
+		loader.IsExternalRefsAllowed = true
+		loader.ReadFromURIFunc = func(loader *kopenapi3.Loader, url *url.URL) ([]byte, error) {
+			var relativePath string
+
+			if strings.HasPrefix(url.Path, "./") {
+				relativePath = strings.TrimRightFunc(key, func(r rune) bool { return r != '/' }) + strings.TrimPrefix(url.Path, "./")
+			} else {
+				relativePath = url.Path
+			}
+
+			file, err := ns.deps.SourceFilesystems.Assets.Fs.Open(relativePath)
+			if err != nil {
+				return nil, err
+			}
+
```

Comment:
Missing `defer file.Close()`

Reply:
just throwing this out there, but it looks like the contributor added the `defer file.Close()` on line `108`